### ISSUE TITLE
Invalid SQL queries for Apache Druid

### ISSUE THREAD

**Issue by adammruk** [cube-js/cube.js#2149]
**Describe the bug**
When using Druid adapter, CubeJS is creating invalid SQL queries, which prevents to use CubeJS with Druid DB.

1. Repetitive error in console:
When connected to Druid database, in CubeJS backend terminal throws this error every ~30s:
```Error querying db: scheduler-c53d5a25-db2b-4b08-a608-608e4363ec84 
{} 
Error: org.apache.calcite.runtime.CalciteContextException: From line 1, column 34 to line 1, column 38: No match found for function signature NOW()
    at QueryQueue.parseResult (<CubeJS backend path>/node_modules/@cubejs-backend/query-orchestrator/src/orchestrator/QueryQueue.js:100:13)
    at QueryQueue.executeInQueue (<CubeJS backend path>/node_modules/@cubejs-backend/query-orchestrator/src/orchestrator/QueryQueue.js:89:19)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at async Promise.all (index 0)
    at OrchestratorApi.executeQuery (<CubeJS backend path>/node_modules/@cubejs-backend/server-core/src/core/OrchestratorApi.ts:40:20)
    at <CubeJS backend path>/node_modules/@cubejs-backend/server-core/src/core/RefreshScheduler.ts:158:9
    at async Promise.all (index 0)
    at <CubeJS backend path>/node_modules/@cubejs-backend/server-core/src/core/RefreshScheduler.ts:143:7
    at async Promise.all (index 1)
```
Which actually seems to be correct error, cause in Druid there's no such query like `NOW()` (for current time, there's `CURRENT_TIMESTAMP` for example)

2. Another SQL error is with creating some example chart. In playground, after select Measure > Wikipedia Count, following error is visible: 
 
![obraz](https://user-images.githubusercontent.com/19703066/108503589-eb29d880-72b4-11eb-8fa5-d9bdc5a89d03.png)

**To Reproduce**
1. Install Apache Druid DB, and load wikipedia example data, as described in https://druid.apache.org/docs/latest/tutorials/index.html
2. Install CubeJS for Druid database: `npx cubejs-cli create cubejs -d druid`
3. Set .env file:
```
CUBEJS_DB_URL="http://localhost:8082"
CUBEJS_DB_USER=""
CUBEJS_DB_PASS=""
```
4. `npm install`
5. `npm run dev`
6. Go to the playground (http://localhost:4000/#/schema), and in "Schema" tab generate Schema:
![obraz](https://user-images.githubusercontent.com/19703066/108504413-1d880580-72b6-11eb-863b-ff89542e30c8.png)
7. Go to "Build" tab
8. From "Measures" select "Wikipedia Count"
9. Error with invalid query is visible in Playground, and in terminal every ~30s error with `NOW()` is visible.

**Expected behavior**
I should be able to use Druid Database without any errors

**Version:**
From `npm list | grep @cubejs-backend`:
```  
├─┬ @cubejs-backend/druid-driver@0.26.16
│ ├─┬ @cubejs-backend/query-orchestrator@0.26.16
│ │ ├── @cubejs-backend/shared@0.26.16 deduped
│ ├─┬ @cubejs-backend/schema-compiler@0.26.16
│ ├─┬ @cubejs-backend/shared@0.26.16
└─┬ @cubejs-backend/server@0.26.16
  ├── @cubejs-backend/dotenv@9.0.2
  ├─┬ @cubejs-backend/server-core@0.26.16
  │ ├─┬ @cubejs-backend/api-gateway@0.26.16
  │ │ ├── @cubejs-backend/shared@0.26.16 deduped
  │ ├── @cubejs-backend/query-orchestrator@0.26.16 deduped
  │ ├── @cubejs-backend/schema-compiler@0.26.16 deduped
  │ ├── @cubejs-backend/shared@0.26.16 deduped
  ├── @cubejs-backend/shared@0.26.16 deduped

  ```

**Additional context**
OS: macOS Big Sur 11.2.1

**Comment by ovr** [comment]
Hello @adammruk,

Thank you for submitting an issue. Good catch, I can approve that Apache Druid doesn't support `NOW()`, I suggest you prepare a PR, it's pretty simple.

You should override `nowTimestampSql` inside `DruidQuery`:

https://github.com/cube-js/cube.js/blob/master/packages/cubejs-druid-driver/src/DruidQuery.ts#L14

```
  public nowTimestampSql(): string {
    return `CURRENT_TIMESTAMP`;
  }
```

Thanks

**Comment by ovr** [comment]
Thank you @adammruk , Released in `v0.26.19`
